biocypher/_config/biocypher_config.yaml

```diff
@@ -131,3 +131,11 @@ sqlite:
   delimiter: '\t'
   # import_call_bin_prefix: '' # path to "sqlite3"
   # import_call_file_prefix: '/path/to/files'
+
+csv:
+  ### CSV/Pandas configuration ###
+  delimiter: ','
+
+networkx:
+  ### NetworkX configuration ###
+  some_config: some_value
```

Comment:
Needed to add at least one dummy value here, otherwise parsing fails. Solves automatically in the future, when config options are added.